### ISSUE TITLE
Normal mapping results in artifacts when both mesh batching and precompute tangents options are enabled

### ISSUE THREAD

**Issue by QuantumCoderQC** [armory3d/armory#2845]
**Description**
When both `Batch Meshes` and `Precompute Tangents` options are enabled, strange artefacts appear on the rendered mesh.

For example, consider a 2 X 2 arrangement of 4 identical grid mesh objects:
![image](https://user-images.githubusercontent.com/55564981/230657552-625f6ed5-2b5d-44dd-8720-749372d7355f.png)

The meshes share the following material setup:
![image](https://user-images.githubusercontent.com/55564981/230657762-0e407000-40c1-45d9-86b6-32f662814097.png)

| Either/ both options disabled | Both options enabled|
|--------|--------|
| Correct: ![image](https://user-images.githubusercontent.com/55564981/230658611-10771298-a71e-4520-9997-6ef37be0ae5f.png)| Incorrect: ![image](https://user-images.githubusercontent.com/55564981/230658669-81ca263b-a7c6-4d84-9112-00fc021c7f7b.png)|

Also, this issue does not happen if all the objects share the same mesh data (like linked duplicates).

**To Reproduce**
Run the provided blend file.

**System**
Blender: 3.3 LTS
Armory: SDK 2023.4
OS: Windows 10
Graphics card: Nvdia 2030

**Test File**
[Mesh Batch Normal Mapping Tangents.zip](https://github.com/armory3d/armory/files/11180767/Mesh.Batch.Normal.Mapping.Tangents.zip)
